src/js/views/hallOfFame.js

```diff
@@ -42,16 +43,32 @@ async function updatePlayers(inputs, updateEvents) {
                 }
             }
 
-            players[i].bestStats = {
-                gp: 0,
-                min: 0,
-                per: 0,
-            };
+            players[i].bestStats = {};
+            let bestEWA = 0;
+            players[i].legacy = {};
             for (let j = 0; j < players[i].stats.length; j++) {
-                if (players[i].stats[j].gp * players[i].stats[j].min * players[i].stats[j].per > players[i].bestStats.gp * players[i].bestStats.min * players[i].bestStats.per) {
+                const rEWA = players[i].stats[j].ewa;
+                const team = players[i].stats[j].abbrev;
+                let pEWA = 0;
+                for (let k = 0; k < players[i].statsPlayoffs.length; k++) {
+                    if (players[i].stats[j].season === players[i].statsPlayoffs[k].season && team === players[i].statsPlayoffs[k].abbrev) {
+                        pEWA = players[i].statsPlayoffs[k].ewa;
+                        break;
+                    }
+                }
+                if (rEWA + pEWA > bestEWA) {
+                    bestEWA = rEWA + pEWA;
                     players[i].bestStats = players[i].stats[j];
                 }
+                if (players[i].legacy.hasOwnProperty(team)) {
+                    players[i].legacy[team] += rEWA + pEWA;
+                } else {
+                    players[i].legacy[team] = rEWA + pEWA;
+                }
             }
+
+            const careerTeam = Object.keys(players[i].legacy).reduce((teamA, teamB) => {return players[i].legacy[teamA] > players[i].legacy[teamB] ? teamA : teamB;});
```

Comment:
Replace `{return players[i].legacy[teamA] > players[i].legacy[teamB] ? teamA : teamB;}` with `players[i].legacy[teamA] > players[i].legacy[teamB] ? teamA : teamB` and through the magic of ES6 it will work (brackets and return are inferred for single line arrow functions).